.devcontainer/docker-compose.yml

```
@@ -0,0 +1,87 @@
+#
+# ---------------------------------------------------------------------
+#
+# GLPI - Gestionnaire Libre de Parc Informatique
+#
+# http://glpi-project.org
+#
+# @copyright 2015-2023 Teclib' and contributors.
+# @copyright 2003-2014 by the INDEPNET Development Team.
+# @licence   https://www.gnu.org/licenses/gpl-3.0.html
+#
+# ---------------------------------------------------------------------
+#
+# LICENSE
+#
+# This file is part of GLPI.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# ---------------------------------------------------------------------
+#
+
+version: '3.9'
+
+services:
+  php:
+    container_name: "glpi-php-fpm"
+    build:
+      context: conf/php
+    restart: unless-stopped
+    volumes:
+      - ..:/var/www/:rw
+      - ./conf/php/xdebug.ini:/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
+    depends_on:
+      - db
+    extra_hosts:
+    - "host.docker.internal:host-gateway"
+
+  nginx:
+    container_name: "glpi-nginx"
+    image: "nginx:alpine"
+    command: ["nginx"]
+    restart: unless-stopped
+    volumes:
+      - ..:/var/www:rw
```

`nginx` is not supposed to write anything.
```suggestion
      - ..:/var/www:ro
```